packages/forma-36-website/src/pages/components/modal/index.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: 'Modal'
+type: 'component'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Modal'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-modal--default'
+---
+
+Modal dialogs interrupt users and demand an action. They are appropriate when user’s attention needs to be directed toward important information.
+
+```jsx
+function ModalExample() {
+  const [isShown, setShown] = React.useState(false);
+  return (
+    <React.Fragment>
+      <Button onClick={() => setShown(true)}>Show centered modal</Button>
+      <Modal title="Centered modal" isShown={isShown}>
+        {() => (
+          <React.Fragment>
+            <Modal.Header title="Title" />
+            <Modal.Content>Hello from controlled modal window</Modal.Content>
+            <Modal.Controls>
+              <Button buttonType="positive" onClick={() => setShown(false)}>
+                Confirm
+              </Button>
+              <Button buttonType="muted" onClick={() => setShown(false)}>
+                Close
+              </Button>
+            </Modal.Controls>
+          </React.Fragment>
+        )}
+      </Modal>
+    </React.Fragment>
+  );
+}
+```
+
+## Table of contents
+
+1. [Best practices](#best-practices)
+2. [Variations](#variations)
+   1. [Position](#position)
+3. [Copy considerations](#copy-considerations)
+4. [Accessibility](#accessibility)
+
+## Best practices <a name="best-practices" />
+
+- Use modal dialogs to request the user to enter information critical to continuing the current process.
```

Comment:
- Use modal dialogs to request information critical for a user to continue their current process
- Use modal dialogs to warn users about a potentially destructive action they are taking
- Use modal dialogs consistently across use cases
- Do not use modal dialogs for nonessential information that is not related to the current user flow
- Avoid modal dialogs that interrupt high-stake processes such as checkout flows